src/Autofac.Multitenant/MultitenantContainer.cs

```diff
@@ -342,7 +342,7 @@ public void ConfigureTenant(object tenantId, Action<ContainerBuilder> configurat
     /// This should happen very rarely, hopefully never.
     /// </summary>
     [SuppressMessage("CA1513", "CA1513", Justification = "ObjectDisposedException.ThrowIf is not available in all target frameworks.")]
-    private ILifetimeScope CreateTenantScope(object tenantId, Action<ContainerBuilder>? configuration = null)
+    private ILifetimeScope CreateTenantScope(object tenantId, Action<ContainerBuilder>? configuration = null, bool forced = false)
```

Comment:
Do we still need the `forced` parameter here? On quick scan it doesn't look like this gets called.

Reply:
I am reverting all the changes